x-pack/plugin/ccr/src/main/java/org/elasticsearch/xpack/ccr/CcrRetentionLeases.java

```diff
@@ -17,8 +17,22 @@
      * @param leaderUUID   the leader index UUID
      * @return the retention lease ID
      */
-    public static String retentionLeaseId(final String followerUUID, final String leaderUUID) {
-        return String.format(Locale.ROOT, "%s-following-%s", followerUUID, leaderUUID);
+    public static String retentionLeaseId(
+            final String localClusterName,
+            final String remoteClusterName,
+            final String followerIndexName,
+            final String followerUUID,
+            final String leaderIndexName,
+            final String leaderUUID) {
+        return String.format(
+                Locale.ROOT,
+                "%s/%s/%s-following-%s/%s/%s",
```

Comment:
should we prefix the retention leases for CCR with `ccr-`. This will allow us to easily filter on them

Reply:
@ywelsch The source on a retention lease for CCR will be `ccr`. Is that sufficient?

Reply:
ok, even better